heartbeat/IPaddr2

```diff
@@ -1002,6 +1003,11 @@ ip_served() {
 
 	if [ -z "$IP_CIP" ]; then
 		for i in $cur_nic; do
+                        # check address label
```

Comment:
I just noticed you're using spaces for indentation, so that needs to be updated to tabs to follow the rest of the code.

Reply:
done